org.claws_mail.Claws-Mail.json

```diff
@@ -0,0 +1,161 @@
+{
+  "app-id": "org.claws_mail.Claws-Mail",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "claws-mail",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=home:rw",
+    "--filesystem=/etc/skel/.claws-mail:ro"
+  ],
+  "modules": [
```

Comment:
You need to add `cleanup` section (global one above or per module) to delete static libs and development headers.

Reply:
Will look into this, but I'm not that familiar with the compiling processes, so there will be some guessing involved. I'll check some of the other Flathub flatpak repo's for examples.